tests/TestCase.php

```diff
@@ -4,6 +4,7 @@
 
 namespace MatchBot\Tests;
 
+use Cassandra\Uuid;
```

Comment:
I think this is unintentional and importing the other Uuid would make the below more concise instead?

Reply:
Yes, will change.

Reply:
Done in 0f5768e8